rpc/handlers.go

```diff
@@ -0,0 +1,40 @@
+package rpc
+
+import (
+	"encoding/hex"
+	"log"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"github.com/pokt-network/pocket/app"
+)
+
+func (s *rpcServer) GetV1Health(ctx echo.Context) error {
+	return ctx.NoContent(http.StatusOK)
+}
+
+func (s *rpcServer) GetV1Version(ctx echo.Context) error {
+	return ctx.String(http.StatusOK, app.AppVersion)
+}
+
+func (s *rpcServer) PostV1ClientBroadcastTxSync(ctx echo.Context) error {
+	params := new(RawTXRequest)
+	if err := ctx.Bind(params); err != nil {
+		return ctx.String(http.StatusBadRequest, "bad request")
+	}
+	bz, err := hex.DecodeString(params.RawHexBytes)
+	if err != nil {
+		return ctx.String(http.StatusBadRequest, "cannot decode tx bytes")
+	}
+	height := s.bus.GetConsensusModule().CurrentHeight()
+	uCtx, err := s.bus.GetUtilityModule().NewContext(int64(height))
+	if err != nil {
+		defer func() { log.Fatalf("[ERROR] Failed to create UtilityContext: %v", err) }()
```

Comment:
Interesting pattern. Does this return before the fatal log is called?

Reply:
https://go.dev/play/p/gY_tFOpV8Vm

Reply:
In Italian "Ciao" is used also to say "Hello". In other languages like  Brazilian Portuguese it's only for "Bye".

It was fun for me working in a Brazilian company for a few years and noticing weird reactions whenever I said "ciao" thinking I was cool. 😂